### PR TITLE
feat(TDI-46441): support cassandra 4.0

### PR DESCRIPTION
**Pull request by chmyga**
**Please check if the PR fulfills these requirements**

- [ ] The commit(s) message(s) follows our [guidelines](https://github.com/talend/tools/blob/master/tools-root-github/CONTRIBUTING.md#commit-message-format) ?
- [ ] Unit tests for the Java changes have been added (for bug fixes / features) ?
- [ ] TUJ for the JavaJet changes have been added (for bug fixes / features) ?
- [ ] Docs have been added / updated (for bug fixes / features) ?
- [ ] The new code does not introduce new technical issues

**What is the current behavior?** (You can also link to an open issue here)

**What is the new behavior?**

**BREAKING CHANGE**

If this PR contains a breaking change, please describe the impact and migration
path for existing applications.
If not please remove this section.

**Other information**:
https://jira.talendforge.org/browse/TDI-46441

### CODE REVIEW

**Review by clesaec — APPROVED** [review]
Should be clearer with something like : 
```java
interface CassandraSession {
    ResultSet execute(String request);
    void close();
}

class CassandraSessionV4 implements  CassandraSession {
    public CassandraSessionV4(Map<String, Object> globalMap) { ...}
    ...
}

class CassandraSessionV3  implements  CassandraSession {
    public CassandraSessionV3(Map<String, Object> globalMap) { ...}
    ...
}
```
At start of tCassandraInput_datastax_begin.javajet (or tCassandraInput_begin.javajet) but that's OK for Javajet ....

File: main/plugins/org.talend.designer.components.bigdata/components/tCassandraBulkExec/tCassandraBulkExec_java.xml [hunk 1]
```diff
@@ -466,6 +564,154 @@
...
+				<IMPORT
+				    NAME="native-protocol-1.5.0"
+				    MODULE="native-protocol-1.5.0.jar" MVN="mvn:com.datastax.oss/native-protocol/1.5.0" />
+				<IMPORT
+				    NAME="reactor-core-3.4.2"
+				    MODULE="reactor-core-3.4.2.jar" MVN="mvn:io.projectreactor/reactor-core/3.4.2" />
+				<IMPORT
+                    NAME="jul-to-slf4j-1.7.26"
+                    MODULE="jul-to-slf4j-1.7.26.jar" MVN="mvn:org.slf4j/jul-to-slf4j/1.7.26" />
+				<IMPORT MODULE="netty-common-4.1.60.Final.jar"
```
Comment (wwang-talend) [inline main/plugins/org.talend.designer.components.bigdata/components/tCassandraBulkExec/tCassandraBulkExec_java.xml:653]: please align version to 4.1.68.Final

**Review by wwang-talend — CHANGES_REQUESTED** [review]

File: main/plugins/org.talend.designer.components.bigdata/components/tCassandraBulkExec/tCassandraBulkExec_java.xml (hunk 1 above)
Comment (dmytro-sylaiev) [reply main/plugins/org.talend.designer.components.bigdata/components/tCassandraBulkExec/tCassandraBulkExec_java.xml]: Done

**Review by jzhao-talend — APPROVED** [review]
LGTM

**Review by wwang-talend — APPROVED** [review]

**Pull request merged**